crates/wasmparser/benches/benchmark.rs

```diff
@@ -230,7 +235,7 @@ fn it_works_benchmark(c: &mut Criterion) {
     let mut inputs = collect_benchmark_inputs();
     // Filter out all benchmark inputs that fail to parse via `wasmparser`.
     inputs.retain(|input| read_all_wasm(input.wasm.as_slice()).is_ok());
-    c.bench_function("it works benchmark", move |b| {
+    c.bench_function("parse all test modules", move |b| {
```

Comment:
I think this identifier is supposed to be without white space usually so that we can filter benchmarks easily. In `wasmi` we namespace benchmarks using `/` for example: `"parse/visit"` and `"parse/read"` or so. With whitespace it becomes daunting to filter benchmark test cases via CLI.

Reply:
I think you can pass regexes to criterion to get it to work regardless, but I prefer `/`-separated tests anyway.